rdbms/src/test/java/io/spine/server/storage/jdbc/record/JdbcRecordStorageShould.java

```diff
@@ -111,8 +111,7 @@ public void read_by_composite_filter_with_column_filters_for_same_column() {
         final EntityFilters entityFilters = EntityFilters.newBuilder()
                                                          .addFilter(columnFilter)
                                                          .build();
-        final EntityQuery<String> query = EntityQueries.from(entityFilters,
-                                                             TestEntityWithStringId.class);
+        final EntityQuery<String> query = EntityQueries.from(entityFilters, storage.getEntityColumnCache());
```

Comment:
I don't like this. Now the users of the API know about the cache existence. 

Let's discuss.